concourse/pipeline.yml

```diff
@@ -114,7 +114,7 @@ jobs:
           CF_SPACE: production
           INSTANCES: 30
           CF_STARTUP_TIMEOUT: 15 # minutes
-          REQUIRE_BASIC_AUTH: "true"
+          REQUIRE_BASIC_AUTH: "false"
```

Comment:
Hmm. In the vulnerable people form, `REQUIRE_BASIC_AUTH` is nil (https://github.com/alphagov/govuk-coronavirus-vulnerable-people-form/blob/master/concourse/pipeline.yml#L123). This is because in that pipeline, [the deploy to PaaS](https://github.com/alphagov/govuk-coronavirus-vulnerable-people-form/blob/master/concourse/tasks/deploy-to-govuk-paas.yml#L43-L45) checks if it's set to true or anything, and sets the envvar if so.

That behaviour seems to be the same [in this pipeline](https://github.com/alphagov/govuk-coronavirus-find-support/blob/master/concourse/tasks/deploy-to-govuk-paas.yml#L43-L45), so do we want to do this:

```suggestion
          REQUIRE_BASIC_AUTH:
```

❓

Reply:
Thanks Issy.  I've updated to remove the value completely.

Reply:
@brucebolt You removed it completely in staging, not production.